pkgs/applications/editors/bluej/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, makeWrapper, jetbrains }:
+
+stdenv.mkDerivation rec {
+  pname = "bluej";
+  version = "4.2.2";
+  src = fetchurl {
+    url = "https://www.bluej.org/download/files/BlueJ-linux-${builtins.replaceStrings ["."] [""] version}.deb";
+    sha256 = "sha256-XCJB8iCOmPz5qtfHooK88W5v1UP6pf2wuZs00QIxE8M=";
+  };
+
+  buildInputs = [ makeWrapper ];
+  propagatedBuildInputs = [ jetbrains.jdk ];
+
+  unpackPhase = ''
+    ar xf $src
+    tar xf data.tar.xz
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp -r usr/* $out
+
+    makeWrapper ${jetbrains.jdk}/bin/java $out/bin/bluej \
+      --add-flags "-Djavafx.embed.singleThread=true -Dawt.useSystemAAFontSettings=on -Xmx512M -cp \"$out/share/bluej/bluej.jar:${jetbrains.jdk}/lib/tools.jar\" bluej.Boot"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A simple integrated development environment for Java";
+    homepage = "https://www.bluej.org/";
+    license = licenses.gpl2;
```

Comment:
See https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md .

Reply:
Fixed, sorry about that.